src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -186,20 +183,6 @@ private void SafeOnBufferChange(object sender, TextContentChangedEventArgs e)
             }
         }
 
-        private void UpdateDirtySpans(TextContentChangedEventArgs e)
-        {
-            currentSnapshot = e.After;
-
-            var newDirtySpans = dirtySpans.CloneAndTrackTo(e.After, SpanTrackingMode.EdgeInclusive);
-
-            foreach (var change in e.Changes)
-            {
-                newDirtySpans = NormalizedSnapshotSpanCollection.Union(newDirtySpans, new NormalizedSnapshotSpanCollection(e.After, change.NewSpan));
-            }
-
-            dirtySpans = newDirtySpans;
-        }
```

Comment:
The original version of this code was copied from the VSSDK [Error List](https://github.com/microsoft/VSSDK-Extensibility-Samples/tree/master/ErrorList) sample.

The sample is for a spelling checker that displays typos in the Error List. It did use the set of dirty spans, as it used them to work out which bits of the document had changed and needed to be spell-checked again (i.e. it effectively re-ran the "analysis" on the just the bits of the document that had changed).